Makefile

```diff
@@ -50,7 +50,7 @@ migrate-db: ## Apply database migrations
 
 .PHONY: db-schema
 db-schema: ## create a dump of current db schema
-	pg_dump --username conch --schema-only --file sql/schema.sql conch
+	@carton exec bin/conch_db dump-schema --file sql/schema.sql
```

Comment:
Heh, I also created a bin/conch_db executable in my branch for #625 (it's much shorter, doing a few utility things, some of which overlap here) :)

Reply:
Yep, I specifically used #625 as the basis for this command, the WIP tag is mostly because I don't have all of the things you do there implemented yet.

Reply:
Yours actually was `bin/conch-db` and so there wasn't a conflict, I just had to port my stuff over. I've since abandoned my own `bin/conch_db` because yours was less invasive.